docker-compose.yml

```diff
@@ -112,7 +112,7 @@ services:
       - SIGMAX_SERVER
       - DWH_MEDIA_ROOT=/dwh_media
       - ELASTICSEARCH_HOST=elasticsearch:9200
-      - INITIALIZE_WITH_DUMMY_DATA=0
+      - INITIALIZE_WITH_DUMMY_DATA=1
```

Comment:
Is this committed with a specific intention? It could give issues if you are not aware it has been set to 1